oscarapi/serializers/checkout.py

```diff
@@ -182,6 +182,7 @@ class Meta:
 class CheckoutSerializer(serializers.Serializer, OrderPlacementMixin):
     basket = serializers.HyperlinkedRelatedField(
         view_name='basket-detail', queryset=Basket.objects)
+    guest_email = serializers.EmailField(allow_blank=True, required=False)
```

Comment:
Why would you add `allow_blank=True`? Now it's possible to do an anonymous checkout without a valid email address